source/guide_wkd.rst

```diff
@@ -0,0 +1,197 @@
+ [isabell@stardust ~]$
+
+Add the following lines to the empty ``.htaccess`` file to set the correct headers:
+
+.. code-block:: apache
+
+ ## WEB KEY DIRECTORY ##
+ <IfModule mod_mime.c>
+    ForceType application/octet-stream
+    Header always set Access-Control-Allow-Origin "*"
+ </IfModule>
+
+.. warning:: Make sure that there is no automatic directory listing! It is not necessary and it reveals the number of emailadresses (and their hashes) in the WKD (this can be a privacy issue)!
+
+
+Direct method
+-------------
+
+Create the directories and symlinks
+
+::
+
+ [isabell@stardust ~]$ cd ~/
+ [isabell@stardust ~]$ mkdir -p html/.well-known
+ [isabell@stardust ~]$ ln -s /var/www/virtual/isabell/openpgpkey.stardust.uberspace.de/.well-known/openpgpkey/stardust.uberspace.de/ /var/www/virtual/isabell/html/.well-known/openpgpkey
+ [isabell@stardust ~]$
+ 
+Create another symlink to facilitate uploading keys via scp:
+
+::
+
+ [isabell@stardust ~]$ ln -s /var/www/virtual/isabell/openpgpkey.stardust.uberspace.de/.well-known/openpgpkey/stardust.uberspace.de/hu/ ./
+ [isabell@stardust ~]$
+
+Upload GPG keys
+===============
+
+Obtaining the WKD-Hashes
+------------------------
+
+The keys are stored in files named by the WKD-Hashes of the Mailuser. Take the prefix of your e-mail address (i.e. in isabell@stardust.uberspace.de, this would be `isabell`), hash it with SHA-1 and then encode the output with z-base-32.
+
+You can get the hashes for all identities of your GPG key with this command:
+
+::
+
+ [someuser@somehost ~]$ gpg --with-wkd-hash --list-public-keys "isabell@stardust.uberspace.de"
+ [...]
+       0123456789ABCDEF0123456789ABCDEF01234567
+ uid           [ unknown] Isabell <isabell@stardust.uberspace.de>
+            mmuhurigesr7z8hzf6sh5cmfsnmiiyyr@stardust.uberspace.de
+ [...]
+ [someuser@somehost ~]$
+
+Alternatively you can use https://cryptii.com/pipes/z-base-32 for that (add a hash-block with SHA-1 before the z.base-32-block).
+
+Summary:
+ * E-Mail: isabell@startdust.uberspace.de
+ * Prefix: isabell
+ * WKD-Hash: mmuhurigesr7z8hzf6sh5cmfsnmiiyyr
+
+
+Exporting a GPG key
+-------------------
+
+::
+
+ [someuser@somehost ~]$ gpg --no-armor --export isabell@stardust.uberspace.de > mmuhurigesr7z8hzf6sh5cmfsnmiiyyr
+ [someuser@somehost ~]$
+ 
+.. note:: The public key is the the same for all identities of a key. You can simply duplicate the exported key, using the corresponding WKD-Hash as filename.
+
+Upload
+------
+
+::
+
+ [someuser@somehost ~]$ scp mmuhurigesr7z8hzf6sh5cmfsnmiiyyr isabell@stardust.uberspace.de:hu/
+ [someuser@somehost ~]$
+
+Testing
+=======
+
+First of all, these two URLs should be available using your browser:
+
+ * Advanced method: https://openpgpkey.stardust.uberspace.de/.well-known/openpgpkey/stardust.uberspace.de/hu/mmuhurigesr7z8hzf6sh5cmfsnmiiyyr
+ * Direct method: https://stardust.uberspace.de/.well-known/openpgpkey/hu/mmuhurigesr7z8hzf6sh5cmfsnmiiyyr
```

Comment:
Users cannot host content on `$host.uberspace.de`, so I guess there is a typo in that URL?

Reply:
I thought "stardust.uberspace.de" is the equivalent to "example.org" to use for the guides. I could use isabell@example.org and openpgpkey.example.org for the guide if you prefer it though.